kasa/smartstrip.py

```diff
@@ -154,6 +155,35 @@ async def set_alias(self, alias: str) -> None:
         """
         return await super().set_alias(alias)
 
+    @requires_update
+    async def get_emeter_realtime(self) -> EmeterStatus:
+        """Retrieve current energy readings."""
+        if not self.has_emeter:
+            raise SmartDeviceException("Device has no emeter")
+
+        plug_emeter = False
+        for plug in self.children:
+            if plug.has_emeter:
+                plug_emeter = True
+
+        if not plug_emeter:
+            return EmeterStatus(await self._query_helper(self.emeter_type, "get_realtime"))
+
+        emeter_rt: DefaultDict[int, float] = defaultdict(lambda: 0.0)
+        count = 0
+        for plug in self.children:
+            if not plug.has_emeter:
```

Comment:
Is this a real-world case, where some sockets on the same strip have and some sockets don't have emeter? If not I think this would be redundant